### PR TITLE
[Ceph-Ansible] ansible changes w.r.t octopus

### PR DESCRIPTION
**Pull request by sunilkumarn417**
- Added ansible changes for RHCS 5.0.
- Added initial octopus directory for configuration and test-suite.

**4.1 - rhel-8.2** : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1594829781660/
**5.0 - rhel-8.2** : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1594788389074/
**5.0 - rhel-8.2 - containerized** : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1594742187385/

Signed-off-by: sunilkumarn417 <sunnagar@redhat.com>

### CODE REVIEW

File: suites/octopus/ansible/sanity_ceph_ansible.yaml [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+      module: test_ansible.py
+      config:
+        is_mixed_lvm_configs: True
+        ansi_config:
+            ceph_test: True
+            ceph_origin: distro
+            ceph_stable_release: octopus
+            ceph_repository: rhcs
+            osd_scenario: lvm
+            journal_size: 1024
```
Comment (XDsubhash) [inline suites/octopus/ansible/sanity_ceph_ansible.yaml:19]: can we remove this journal?

Comment (XDsubhash) [reply suites/octopus/ansible/sanity_ceph_ansible.yaml:19]: everywhere in this PR under ansi_config:

**Comment by sunilkumarn417** [comment]
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1594982640749/

File: suites/octopus/ansible/sanity_containerized_ceph_ansible.yaml [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+              ceph_test: True
+              ceph_origin: distro
+              ceph_stable_release: octopus
+              ceph_repository: rhcs
+              osd_scenario: lvm
+              osd_auto_discovery: False
+              ceph_stable: True
+              ceph_stable_rh_storage: True
+              ceph_docker_image: rhceph/rhceph-3-rhel7
+              ceph_docker_image_tag: latest
```
Comment (XDsubhash) [inline suites/octopus/ansible/sanity_containerized_ceph_ansible.yaml:23]: probably in another pr these can be removed. once rhcs5.x GA can add them backwith proper creds.

everything else LGTM

**Pull request merged**